### PR TITLE
When no timetable is available and no external link has been set, the copy is incorrect

### PR DESCRIPTION
**Pull request by Coenego**
screen shot 2015-04-07 at 11 55 11

If no external link was added the copy shouldn't include the click on the link below.

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

File: shared/gh/partials/empty-timetable.html [hunk 1]
```diff
@@ -3,7 +3,9 @@
     <i class="fa fa-share-square-o"></i>
 </div>
 <p class="gh-empty-main">Unfortunately this is not in Timetable yet</p>
-<p class="gh-empty-description">Click below to access it, and ask you departmental administrator about making it available via Timetable</p>
 <% if (data.metadata.external) { %>
+    <p class="gh-empty-description">Click below to access it, and ask your departmental administrator about making it available via Timetable</p>
     <a href="<%- data.metadata.external %>" target="_blank" id="gh-empty-access" class="btn btn-default">Access timetable</a>
```
Comment (bp323) [inline shared/gh/partials/empty-timetable.html]: We'll need to be a bit more clever about how to add the link here. Links without `http://` wonn't link properly but look like `http://2014.timetable.cam.ac.uk/www.google.co.uk`

**Comment by bp323** [comment]
Back to @Coenego for followup

File: apps/timetable/ui/js/index.js [hunk 2]
```diff
@@ -196,6 +196,28 @@ define(['gh.core', 'gh.constants', 'gh.subheader', 'gh.calendar', 'gh.student-li
         });
     };
 
+    /**
+     * Open the external link
+     *
+     * @return {Boolean}    Return false to discard the natural link behaviour
+     */
+    var openExternalLink = function() {
```
Comment (bp323) [inline apps/timetable/ui/js/index.js]: Could we reverse the flow a bit? We can call this function and sanitise the URL before rendering the template so when the click happens it's all default behaviour.

**Comment by bp323** [comment]
Back to @Coenego

**Comment by Coenego** [comment]
Back to @bp323 for review after follow-up

**Comment by bp323** [comment]
Verified and merged.

**Pull request merged**